filters/named-entity-count/filter.py

```diff
@@ -0,0 +1,85 @@
+import operator
+from typing import List
+
+import spacy
+
+from initialize import spacy_nlp
+from interfaces.SentenceOperation import (
+    SentenceAndTargetOperation,
+    SentenceOperation,
+)
+from tasks.TaskTypes import TaskType
+
+
+class NamedEntityCountFilter(SentenceOperation):
+    """
+    A filter on the count of named entities in a sentence, with different arithmetic operators.
+    """
+
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(self, op: str = ">", threshold: int = 10):
```

Comment:
One Question, Why the default threshold value is 10?

Reply:
thanks, the default should have been >= 1. fixed it.